cli/wallet/nep17.go

```diff
@@ -580,6 +582,12 @@ func signAndSendNEP17Transfer(ctx *cli.Context, c *client.Client, acc *wallet.Ac
 			return cli.NewExitError(err, 1)
 		}
 	} else {
+		if !ctx.Bool("force") {
```

Comment:
Don't we want to ask confirmation for smartcontract's `deploy` and `invokefunction` and for validator commands?